package/utils/get_style_rule.js

```diff
@@ -22,6 +22,7 @@ const getStyleRule = (test, modules = false, preprocessors = []) => {
       options: {
         sourceMap: true,
         importLoaders: 2,
+        localIdentName: isProduction ? undefined : '[name]__[local]___[hash:base64:5]',
```

Comment:
I guess we can make this a default behavior, even in production and perhaps provide an easier setting to change it, but this is good enough for this PR.